include/seqan3/core/tuple_utility.hpp

```diff
@@ -6,182 +6,14 @@
 // -----------------------------------------------------------------------------------------------------
 
 /*!\file
- * \brief Provides utility functions for tuple like interfaces.
+ * \brief [DEPRECATED] Provides utility functions for tuple like interfaces.
  * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ * \deprecated This header is deprecated and will be removed in SeqAn-3.1.0; Please \#include <seqan3/utility/tuple_utility.hpp> instead.
  */
```

Comment:
```suggestion
 * \deprecated This header is deprecated and will be removed in SeqAn-3.1.0; Please \#include 
               <seqan3/utility/tuple_utility.hpp> instead.
 */
```
You can do line breaks for the doxygen commands.


But, the `SEQAN3_DEPRECATED_HEADER` preprocessor-macro below cannot have line breaks in its message, otherwise it's not displayed correctly.